src/fsearch_window_actions.c

```diff
@@ -26,6 +26,7 @@
 
 #include <glib/gi18n.h>
 #include <stdint.h>
+#include <gdk/gdkx.h>
```

Comment:
Only include when `GDK_WINDOWING_X11` is defined

Reply:
![image](https://github.com/cboxdoerfer/fsearch/assets/60250462/3d491ec1-876d-4981-be7d-a7b0ec05d8de)

`GDK_WINDOWING_X11` is defined only when `x11_enable`  is true

Reply:
'GDK_WINDOWING_X11' is defined in 'gdkconfig.h'.  However, this file can only be directly included with '<gdk/gdk.h>'.  I would need to include '<gdk/gdk.h>' to achieve what you said, which doesn't seem necessary.

Of course, if you think this is very necessary, I can modify it.

Reply:
The `#ifdef` is necessary because otherwise the build fails on systems where GDK was built without the X11 backend.